src/assets/supportMatrix.json

```diff
@@ -480,7 +480,7 @@
     "gmail-android": true
   },
   "width": {
-    "outlook": "Width for p and div tags is not supported",
+    "outlook": true,
```

Comment:
You should create `customSupportMatrix.json` for replacing some properties by merging.